README.md

```diff
@@ -17,12 +17,12 @@ the fastest Machine Intelligence Frameworks in the world.
 
 > See more Deep Neural Networks benchmarks on [Deep Learning Benchmarks][deep-learning-benchmarks-website].
 
-Leaf is portable. Run it on CPUs, GPUs, FPGAs on machines with an OS or on
+Leaf is portable. Run it on CPUs, GPUs, FPGAs, machines with an OS, or
```

Comment:
Those are supposed to be 2 lists "CPUs, GPUs, FPGA" and "with OS, without OS". That change sounds to me like it is one list, but I agree that the old one isn't quite right either.